hudi-common/src/test/java/org/apache/hudi/common/table/log/HoodieLogFormatTest.java

```diff
@@ -78,6 +80,8 @@
 @RunWith(Parameterized.class)
 public class HoodieLogFormatTest {
 
+  @Rule
```

Comment:
+1 on using the `@Rule` wondering if that will give us auto deletion.. I think it's deleted in afterClass of the test. https://stackoverflow.com/questions/16494459/why-isnt-junit-temporaryfolder-deleted ?

Reply:
Yes